### PR TITLE
Jnci edits 10302018

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
@mikeng13 , @thomaszurkan-optimizely : Kindly review and merge if approved.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,345 @@
...
+# Optimizely Isomorphic React Web App Demo SDK Tutorial
+
+This tutorial enables you to quickly get started in your development efforts to create a React/Redux-based webpage with the Optimizely Isomorphic React Web App Demo. This demo package includes an interactive project that contains two experiments. The first experiment illustrates how an online retailer could use a Variation to sort the items to be rendered by price or name. The second experiment shows how a Variation can be used to select between a one-screen or two-screen checkout process.
+
+![test-app screen](./public/main_screenshot.png)
+
+This demo node web app uses the Optimizely Node SDK and JavaScript SDK for A/B Testing. The isomorphic web app is built using React/Redux. All React components are shared between the server and client, and can be rendered both on the server side and the frontend.
+
+The demo works as follows:
+* Configuration starts by setting up a project, feature, and event in the Optimizely dashboard, which will be queried by the demo web page using the Optimizely JavaScript API.
```
Comment (aliabbasrizvi) [inline README.md:10]: This statement states that feature is set up. The rest of the document does not talk about setting up a feature. May be revisit this section.

**Review by aliabbasrizvi — APPROVED** [review]
Looks good overall.

**Review by mikeproeng37 — APPROVED** [review]
lgtm

**Comment by mikeproeng37** [comment]
@jamesneimanconsulting there is a merge conflict with the `README.md` file. Can you please resolve it so we can merge it in?

**Pull request closed without merging**